docs/faq.md

```diff
@@ -48,3 +44,12 @@ Nei, vi støtter kun fødselsnummer som identifikator for en bruker.
 
 ## Hva er retention-tiden på brukernotfikasjon-topicene?
 Alle brukernotifikasjons-topic-ene har uendelig retention.
+
+## Kan det sendes varsel på SMS og/eller e-post til bruker samtidig som det opprettes Brukernotifikasjon på Ditt NAV?
+Ja. Se mer informasjon om dette under [Ekstern varsling](./eksternvarsling.md)
+
+## Kan vi sette vår egen tekst i varsel på SMS og e-post?
+Nei. Det sendes en pre-definert standard tekst, som ikke avslører noen sensitiv informasjon om hva brukerens varsel gjelder. Ta kontakt med oss dersom ditt team har et spesielt behov for å kunne sette deres egen varseltekst.
```

Comment:
Kan hende dere har snakket om dette allerede 🙈  Men vil vi åpne opp for dette? Jeg er bare redd den siste setningen kommer til å trigge flere produsenter. Tror de kommer til å ta kontakt hvis det er krise uansett 🙊  tihi

Reply:
Ja, enig i det egentlig. Er det krise, kommer vi til å høre det.. Jeg tar den vekk.